internal/cmd/asyncapi/command_export.go

```diff
@@ -117,11 +118,11 @@ func (c *command) export(cmd *cobra.Command, _ []string) (err error) {
 					currentSubject: subject,
 				}
 				err := c.getChannelDetails(accountDetails, flags)
-				if err != nil && err == errors.New("protobuf is not supported") {
-					log.CliLogger.Info("Protobuf is not supported.")
-					continue
-				}
 				if err != nil {
+					if err.Error() == protobufErrorMessage {
+						log.CliLogger.Info("Protobuf is not supported.")
```

Comment:
Why not log the error? It's the same text.

Reply:
Using log.CliLogger.log() ?

Reply:
Yeah: `log.CliLogger.Info(err.Error())`